cookbooks/bcpc/templates/default/carbon.conf.erb

```diff
@@ -92,6 +109,32 @@ USE_FLOW_CONTROL = True
 # By default, carbon-cache will log every whisper update. This can be excessive and
 # degrade performance if logging on the same volume as the whisper data is stored.
 LOG_UPDATES = False
+LOG_CACHE_HITS = False
```

Comment:
This `carbon.conf` file has become excruciatingly long for someone to quickly check what the current settings are. A better approach would be to put all the details and possible settings in a `carbon.conf.example` template and only put the values we are setting in `carbon.conf`